benchmark/benchmark_utils.hpp

```diff
@@ -561,7 +561,7 @@ struct bench_naming
     static std::string format_name(std::string string)
     {
         format     format = get_format();
-        std::regex r("([A-z0-9]*):\\s*((?:custom_type<[A-z0-9,]*>)|[A-z:().<>\\s0-9]*)(}*)");
+        std::regex r("([A-z0-9]*):\\s*((?:custom_type<[A-z0-9,]*>)|[A-z:().<>\\s0-9]*)(\\}*)");
```

Comment:
Weird that the behavior is different. Good fix :+1: 
Now I am wondering it is also safer to do the following:
```suggestion
        std::regex r("([A-z0-9]*):\\s*((?:custom_type<[A-z0-9,]*>)|[A-z:\\(\\)\\.<>\\s0-9]*)(\\}*)");
```

Reply:
Sure, might as well - no guarantee the regex will stay valid in the future without escaping the parentheses too.